### DISCUSSION TITLE
Confusion on page 423

### DISCUSSION THREAD

**Discussion by baifanhorst** [rasbt/machine-learning-book#119] [General]
At the bottom of this page, it says that the model has 100% accuracy on the training set, and 95% accuracy on the validation set. However, the left panel of Fig13.4 shows that the model's accuracy on the training set is about 90% and 85% on the validation set. So there is a discrepency between the figure and the description in the text.

**Comment by baifanhorst** [comment]
I tried several times. The model in the book could give the described accuracy, but not always. There is some stochasticity in the result. Not a big issue, though.

**Comment by rasbt** [comment]
Good observation, I just double-checked the TensorFlow edition from 2019, where I got better results. I must have forgotten to update the text here. I will add this as a note to the errata. The interesting question is also why the PyTorch accuracies are lower. Probably not optimal hparam settings.


<img width="583" alt="Screenshot 2025-03-15 at 5 16 20 PM" src="https://github.com/user-attachments/assets/61396713-a7c3-4ff9-ab76-985f2587c8c7" />
